src/main/java/ua/dymohlo/dao/JDBCUserDao.java

```diff
@@ -0,0 +1,159 @@
+            preparedStatement.setString(2, user.getProfession());
+
+            preparedStatement.executeUpdate();
+
+            ResultSet generatedKeys = preparedStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                user.setUsersId(generatedKeys.getInt("users_id"));
+            }
+            return user;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public User findById(Long id) {
+        String GET_SQL = "SELECT users_id, full_name, profession FROM daopractic.users WHERE users_id = ?";
+        try (Connection connection = ConnectorManager.Connection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(GET_SQL);
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                User user = new User(
+                        resultSet.getInt("users_id"),
+                        resultSet.getString("full_name"),
+                        resultSet.getString("profession")
+                );
+                return user;
+            }
+            return null;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+//    @Override
+//    public User update(Long id) {
+//        User user = new User();
+//        String UPDATE_SQL = "UPDATE daopractic.users SET full_name= ?, profession= ? WHERE users_id = ?";
+//        String GET_SQL = "SELECT users_id, full_name, profession FROM daopractic.users WHERE users_id = ?";
+//        try (Connection connection = ConnectorManager.Connection()) {
+//            try (PreparedStatement selectStatement = connection.prepareStatement(GET_SQL)) {
+//                selectStatement.setLong(1, id);
+//                ResultSet resultSet = selectStatement.executeQuery();
+//
+//                if (resultSet.next()) {
+//                    System.out.println("User with id " + id + " does not exist.");
+//                    return null;
+//                }
+//            }
+//            String newFullName = input("Enter new user's full name: ");
+//            String newProfession = input("Enter new user's profession:");
+//            PreparedStatement preparedStatement = connection.prepareStatement(UPDATE_SQL);
+//            preparedStatement.setString(1, newFullName);
+//            preparedStatement.setString(2, newProfession);
+//            preparedStatement.setLong(3, id);
+//
+//            int updateUser = preparedStatement.executeUpdate();
+//            if (updateUser > 0) {
+//                return new User( user.getUsersId(),newFullName, newProfession);
+//            }
+//            return null;
+//        } catch (SQLException e) {
+//            throw new RuntimeException(e);
+//        }
+//    }
+
+    @Override
+    public User update(Long id) {
```

Comment:
Here we should pass object, and set into prepared values data from object we pass to method.